### DISCUSSION TITLE
Hey i need help on a github pages site

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37691] [Other Feature Feedback, Questions, & Ideas] [answered]
so how do you get a github site page domain that looks like `https://thing.github.io` and not `thing.github.io/github.io`

**Comment by radar-blip** [comment, marked as answer]
You can use your username to create the site. So you would use:  `lompycat565.github.io`

Here is some info: https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

**Reply by ghost** [reply]
thanks

**Reply by radar-blip** [reply]
Anytime
